### PR TITLE
expanded a bit to transition into the tutorial to provide a better clarity

### PR DESCRIPTION
**Pull request by joeyouss**
Checklist:

- [x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [x ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

File: client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md
Comment (RandellDawson) [inline client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md:9]: I am fine with this change.

File: client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md
Comment (RandellDawson) [inline client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md:10]: But I think this change is not needed:
```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@joeyouss See my suggested change.

File: client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md
Comment (joeyouss) [reply client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md:10]: Ok thanks for suggestion!

Comment (RandellDawson) [reply client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md:10]: @joeyouss  Can you click the `Commit suggestion` to commit my suggestion.  Thanks.

**Comment by RandellDawson** [comment]
@joeyouss All you need to do is click the `Commit suggestion` button below my last suggestion.

File: client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md
Comment (joeyouss) [reply client/src/pages/learn/machine-learning-with-python/how-neural-networks-work/index.md:10]: Sure I will do that! Thanks for being helpful. It is great for beginners like me. Appreciate it!

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
@moT01 @nhcarrigan @Sky020 Need a second approval before merging.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by Sky020** [comment]
@joeyouss , for future contributions, I recommend taking a look at the [Contribution Docs](https://contribute.freecodecamp.org/#/index) - particularly the suggested PR and commit naming convention.

**Comment by joeyouss** [comment]
Sure! I will make sure of it. Thank you!

**Pull request merged**
